APE0.rst

```diff
@@ -0,0 +1,393 @@
+publicly on an appropriate Project-wide communication channel, and two other
+active Voting Members second the proposal.
+
+For Coordination Committee Member(s) to be removed, two-thirds of the active
+`Voting Members <votingmembers>`_ must vote in support of the removal.
+
+To avoid repeated removal votes, any individual `Voting Member <votingmembers>`_
+who has called for, or seconded such a vote, may not call for or second a vote
+to remove any other Coordination Committee member for one year from the original
+motion.
+
+The Ombudsperson
+----------------
+The Ombudsperson represents the interests of the Astropy community by providing
+an alternative point of contact for sensitive issues such as code of conduct
+violations and ethical concerns. Candidates for this project role are publicly
+nominated by the Coordination Committee, after which the Coordination Committee
+allows at least two weeks for comment, and then the nominee must be confirmed by
+two-thirds of the active `Voting Members <votingmembers>`_.
+
+The Ombudsperson has no term limit but can resign at any time, or be removed by
+the same process as being confirmed: the Coordination Committee initiates, there
+is a two-week comment period, and two-thirds of the active
+`Voting Members <#votingmembers>`_ have to approve the removal. In the period
+between removal/resignation and new appointment, the Coordination Committee will
+temporarily take over the responsibilities of the Ombudsperson, should the need
+arise.
+
+
+.. _votingmembers:
+
+The Voting Members
+------------------
+
+Role
+^^^^
+The Voting Members are the group of trusted individuals who operate the Astropy
+Project on behalf of the community. They have authority over the Astropy
+Project’s technical resources, including the Astropy Project website itself, the
+Astropy GitHub organization and repositories, the issue tracker, and all Astropy
+communication channels. In practice, much of this authority is passed to the
+Coordination Committee via the voting processes described in this Charter.
+
+They also assume many roles required to achieve the Project's goals, especially
+those that require a high level of trust. Collectively, they make decisions that
+shape the future of the Project.
+
+Voting Members are expected to act as role models for the community and
+custodians of the Project, on behalf of the community and all those who rely on
+Astropy. They will act as representatives of the Astropy Project, where
+necessary, including in online discussions or at official Astropy events.
+
+Responsibilities and Rights
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Voting Members may and should participate in formal votes on:
+
+1. Adding new Voting Members
+2. Removing Voting Members
+3. Electing Coordination Committee members
+4. Removing Coordination Committee members
+5. Appointment or removal of the Ombudsperson
+6. Changes to this Charter
+7. Other matters for which the Coordination Committee believes a vote is
```

Comment:
```suggestion
1. Adding new Voting Members
2. Electing Coordination Committee members
3. Appointment of the Ombudsperson
4. Removing Voting Members
5. Removing Coordination Committee members
6. Removal of the Ombudsperson
7. Changes to this Charter
8. Other matters for which the Coordination Committee believes a vote is
```
Looking more closely, I think this is a safer bet.  I didn't like the idea of combining 1/2 and 3/4 because I think we want to make it clear that adding/electing are "normal", and removal are "extraordinary" events.  So I've reordered to sort of reflect that.  Look ok to you, @pllim?